apps/settings/test/unit/panels/screen_lock_passcode/screen_lock_passcode_test.js

```diff
@@ -187,7 +202,7 @@ suite('ScreenLockPasscode > ', function() {
             charCode: 48,
             keyCode: 0,
             preventDefault: function() {}
-          });
+          }).then(() => { done(); });
```

Comment:
I did not realize this was in the setup function. Usually if the function to be tested returns a promise, we call it in the test function. That way we can make sure the function resolves/rejects as expected.